lib/pointing_party_web/live/card_live.ex

```diff
@@ -0,0 +1,114 @@
+defmodule PointingPartyWeb.CardLive do
+  use Phoenix.LiveView
+
+  alias PointingParty.{Card, VoteCalculator}
+  alias PointingPartyWeb.{Endpoint, Presence}
+
+  def render(assigns) do
+    Phoenix.View.render(PointingPartyWeb.CardView, "index.html", assigns)
+  end
+
+  def mount(%{username: username}, socket) do
+    Endpoint.subscribe("users")
+    {:ok, _} = Presence.track(self(), "users", username, %{points: nil})
+    assigns = [
+      current_card: nil,
+      is_driving: false,
+      outcome: nil,
+      party_has_started: false,
+      remaining_cards: [],
+      results: nil,
+      username: username,
+      users: Presence.list("users"),
+      votes: []
+    ]
+
+    {:ok, assign(socket, assigns)}
+  end
+
+  def handle_event("next_card", points, socket) do
+    updated_socket = save_vote_next_card(points, socket)
+    Presence.update(self(), "users", updated_socket.assigns.username, &Map.put(&1, :points, nil))
+    Endpoint.broadcast_from(self(), "users", "update_card", %{card: updated_socket.assigns.current_card})
+
+    {:noreply, updated_socket}
+  end
+
+  def handle_event("start_party", _, socket) do
+    [current_card | remaining_cards] = Card.cards()
+    Endpoint.broadcast("users", "party_started", %{card: current_card})
+
+    {:noreply, assign(socket, is_driving: true, remaining_cards: remaining_cards)}
```

Comment:
Why assign `:remaining_cards` here but wait till the LiveView recieves the broadcast in order to assign `:current_card`?

Reply:
I had done it this way because I was working with the concept of a driver. Only the driver needed to know which cards were remaining. Everyone needed to know about the current card so they could display it in the UI. Let me see if I can remove the driver concept easily. If I can, we should update this code.